Styling/schema/style.meta.schema.json

```diff
@@ -0,0 +1,11 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "meta",
+    "type" : "object",
+    "description" : "A series of property names and the expression to evaluate for the value of each property.",
+    "properties" : {
+    },
+    "additionalProperties" : {
+        "$ref" : "style.metaProperty.schema.json"
```

Comment:
I think we would still need the `style.expression.schema.json` because it can evaluate to anything, which is different from `booleanExpression` and `colorExpression`.